.github/scripts/combine_results.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+
+if [[ "${result}" == "" ]]; then
+  result="[]"
+fi
+
+if [[ "${result_scheduled}" == "" ]]; then
+  result_scheduled="[]"
+fi
+
+# Combine results into new output
+result=$(python -c "import json; print(json.loads('${result}') + json.loads('${result_scheduled}'))")
```

Comment:
That's a slick way to do that, will have to remember this.

Reply:
and when you say slick - you mean haaack lol

Reply:
LoL, I mean, can't it be both? 😁
If jq isn't there, that's a great low-dependency way to do it, that's more reliable than anything else I can think of for handling a hunk of json on a minimal linux.